scripts/get_tool_dl_url.py

```diff
@@ -161,8 +162,21 @@ def get_latest_assets(repo: str) -> Iterable[ReleaseAsset]:
 
     body = response.json()
 
-    for asset in body["assets"]:
-        yield ReleaseAsset.from_json(asset)
+    highest_version = semver.VersionInfo(0, 0, 0)
+    latest_release = None
+
+    for release in body:
```

Comment:
Ah sorry, I have made a mistake by wrong indentation.
The yield should happen outside of `for release in body` loop.

Reply:
Ok modified with using:
  https://python-semver.readthedocs.io/en/latest/usage/check-valid-semver-version.html

This will prevent other non-valid strings, because we do not want to hit problem again when someone will came up with yet another label which is not valid semver.